### PR TITLE
Docs refactoring

### LINKED ISSUE #571: Move contributing guide to docs
We have a file `CONTRIBUTING.md`.

Remove it and instead create Page in Contribution > General with same content

### PR DESCRIPTION
**Pull request by untael**
closes #571 #490 
fixed generators for vue-cli-docs
revert workspaces to make vue-cli-docs works properly with vuestic-ui

### CODE REVIEW

File: packages/ui/src/components/vuestic-components/va-button/VaButton.vue [hunk 1]
```diff
@@ -223,7 +223,7 @@ export default class VaButton extends mixins(
       computedStyle.boxShadow = this.shadowStyle
     }
 
-    if (this.context.va.color && !this.outline && !this.flat) {
+    if (this.context?.va?.color && !this.outline && !this.flat) {
```
Comment (asvae) [inline packages/ui/src/components/vuestic-components/va-button/VaButton.vue:226]: can you use `useColor`? Or whatever we need to interact with color more directly.
va.color more intended for users.

File: packages/ui/src/components/vuestic-components/va-config/VaConfig.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ export const LocalConfigKey = Symbol('LocalConfigKey')
 export const CONFIGS_DEFAULT = []
 
 export function useLocalConfig (): ComponentConfig[] {
-  return inject(LocalConfigKey, CONFIGS_DEFAULT /* TODO This doesn't look correct! */)
+  return inject(LocalConfigKey, CONFIGS_DEFAULT) || []
```
Comment (asvae) [inline packages/ui/src/components/vuestic-components/va-config/VaConfig.ts:36]: > CONFIGS_DEFAULT) || []

Maybe use so

File: packages/vue-cli-docs/src/assets/main.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
-@import "~vuestic-ui-dev/src/components/vuestic-sass/global/reset.scss";
-@import "~vuestic-ui-dev/src/components/vuestic-sass/vuestic-styles.scss";
-@import "~vuestic-ui-dev/src/components/vuestic-sass/icons/icon-fonts/icon-fonts.scss";
-@import "~vuestic-ui-dev/src/components/vuestic-sass/icons/icon-fonts/vuestic-icons/vuestic-icons.scss";
+@import "~vuestic-ui/src/components/vuestic-sass/global/reset.scss";
+@import "~vuestic-ui/src/components/vuestic-sass/vuestic-styles.scss";
+//@import "~vuestic-ui/src/components/vuestic-sass/icons/icon-fonts/icon-fonts.scss";
```
Comment (asvae) [inline packages/vue-cli-docs/src/assets/main.scss:3]: Remove alltogether

File: packages/vue-cli-docs/src/components/DocsNavigation.vue [hunk 4]
```diff
@@ -46,13 +63,14 @@ export default {
           icon: 'fa fa-github',
           url: `https://github.com/epicmaxco/vuestic-ui/tree/develop/packages/docs/examples/${this.gitUrl}.vue`,
         },
-        {
-          text: 'Open in CodePen',
-          icon: 'fa fa-codepen',
-          url: '#',
-        },
+        // {
```
Comment (asvae) [inline packages/vue-cli-docs/src/components/DocsNavigation.vue:66]: remove?

File: packages/vue-cli-docs/src/components/DocsSubtitle.vue [hunk 5]
```diff
@@ -1,16 +1,16 @@
 <template>
 <h3>
   <MarkdownView tag="span" inline :value="$t(text)" />
-  <a :id="anchor" :style="{ color: themes.primary }" :href="`#${anchor}`">#</a>
+  <a :id="anchor" :style="{ color: 'primary' }" :href="`#${anchor}`">#</a>
```
Comment (asvae) [inline packages/vue-cli-docs/src/components/DocsSubtitle.vue:4]: color won't work, no?

File: packages/vue-cli-docs/src/components/header/components/HeaderSelector.vue [hunk 6]
```diff
@@ -36,21 +36,21 @@ export default {
 
   computed: {
     themes () {
-      return this.getColors()
+      return this.getTheme()
```
Comment (asvae) [inline packages/vue-cli-docs/src/components/header/components/HeaderSelector.vue:39]: rename to getColors

File: packages/vue-cli-docs/src/components/header/components/LanguageDropdown.vue [hunk 7]
```diff
@@ -13,13 +13,13 @@
       :class="{ active: option.code === currentLanguage }"
       @click="setLanguage(option.code)"
     >
-      <va-list-item-section :style="{color: themes.primary}">
+      <va-list-item-section :style="{color: 'primary'}">
```
Comment (asvae) [inline packages/vue-cli-docs/src/components/header/components/LanguageDropdown.vue:16]: same here with color

File: packages/vue-cli-docs/src/main.ts [hunk 8]
```diff
@@ -1,14 +1,12 @@
 import { createApp } from 'vue'
 import App from './App.vue'
 import router from './router'
-import store from './store'
 import { i18n } from 'vue-lang-router'
 
-import { VuesticPlugin } from 'vuestic-ui'
-import 'vuestic-ui/dist/vuestic-ui.css'
+import { VuesticPlugin } from 'vuestic-ui/src/main'
+// import 'vuestic-ui/dist/vuestic-ui.css'
```
Comment (asvae) [inline packages/vue-cli-docs/src/main.ts:7]: remove?

**Comment by asvae** [comment]
Found some small issues, but overall looking good.

File: packages/vue-cli-docs/src/main.ts (hunk 8 above)
Comment (untael) [reply packages/vue-cli-docs/src/main.ts:7]: removed

File: packages/vue-cli-docs/src/components/DocsNavigation.vue (hunk 4 above)
Comment (untael) [reply packages/vue-cli-docs/src/components/DocsNavigation.vue:66]: removed

File: packages/vue-cli-docs/src/assets/main.scss (hunk 3 above)
Comment (untael) [reply packages/vue-cli-docs/src/assets/main.scss:3]: removed

File: packages/vue-cli-docs/src/components/DocsNavigation.vue (hunk 4 above)
Comment (untael) [reply packages/vue-cli-docs/src/components/DocsNavigation.vue:66]: removed

File: packages/ui/src/components/vuestic-components/va-config/VaConfig.ts (hunk 2 above)
Comment (untael) [reply packages/ui/src/components/vuestic-components/va-config/VaConfig.ts:36]: fixed

File: packages/vue-cli-docs/src/components/header/components/HeaderSelector.vue (hunk 6 above)
Comment (untael) [reply packages/vue-cli-docs/src/components/header/components/HeaderSelector.vue:39]: fixed

File: packages/vue-cli-docs/src/components/DocsSubtitle.vue (hunk 5 above)
Comment (untael) [reply packages/vue-cli-docs/src/components/DocsSubtitle.vue:4]: fixed

File: packages/vue-cli-docs/src/assets/main.scss (hunk 3 above)
Comment (untael) [reply packages/vue-cli-docs/src/assets/main.scss:3]: removed

File: packages/ui/src/components/vuestic-components/va-config/VaConfig.ts (hunk 2 above)
Comment (untael) [reply packages/ui/src/components/vuestic-components/va-config/VaConfig.ts:36]: fixed

File: packages/vue-cli-docs/src/main.ts (hunk 8 above)
Comment (untael) [reply packages/vue-cli-docs/src/main.ts:7]: removed

File: packages/vue-cli-docs/src/components/header/components/LanguageDropdown.vue (hunk 7 above)
Comment (untael) [reply packages/vue-cli-docs/src/components/header/components/LanguageDropdown.vue:16]: fixed

**Pull request merged**